### ISSUE TITLE
[QUESTION / SUGGESTION] Finding things in space (which is very big)

### ISSUE THREAD

**Issue by Rhedd-V** [tsunamayo/Starship-EVO#2737]
I was happily building my own space station, until I realized that if I flew away I’d never find it again!

The included space stations and ringworld have target brackets that show up at any distance, allowing them to be found. Am I missing a way to do that on my own vessels and stations?

If I’m not overlooking an existing method, adding one REALLY needs to be a priority. Otherwise, how can you ever find a custom station? How can fighters ever find their carrier? Anything further than 1km away is probably gone forever. Without some way of locating  things, players can’t expand the universe with their own creations.

Just having easy access to spatial coordinates would help, but I would also suggest a “transponder” brick, which could either have a range set to show up on sensors, or have it show up in a greater range as the block is stretched/stacked to be larger.

**Comment by ProPeach** [comment]
Press F3 which will bring up the debug menu, then scroll right to the bottom and tick the "allow teleport to all ships" option.
Then open to codex and click on the teleport menu (on the left) and select your ship.
Normally you can teleport to all ships within range if they have a teleporter block on them if that debug option isn't checked.

**Comment by Rhedd-V** [comment]
Thank you for the information, but teleporting doesn’t solve the problem when you want to pilot a ship to rendezvous with a distant ship.

Is there some way to mark a creation as a station, so it will show up at any distance? I haven’t found one, which makes the included stations unique, and player-built stations practically impossible.

**Comment by tsunamayo** [comment]
Yes it is on the to-do list for stations - I am not fixed on the exact form it will takes though. Might be an antenna or a station block...

**Comment by Rhedd-V** [comment]
Glad it’s on the list!

Just please make whatever system you choose work on ships as well as stations, so you can locate a carrier/mothership at a distance.

 (An antenna piece would be cool)
